DefensiveItem.cpp

```diff
@@ -1,12 +1,13 @@
 #include "DefensiveItem.h"
-#include "Dwarf.h"
-#include "Paladin.h"
-#include "DragonSlayer.h"
-#include "Dragon.h"
-#include <cassert>
+#include "Utility.h"
+// #include "Dwarf.h"
+// #include "Paladin.h"
+// #include "DragonSlayer.h"
+// #include "Dragon.h"
```

Comment:
Delete dead code.
There's no point leaving it in.  If you need to see it again, you can dig through the commit history to see it.
- [ ] revise